tests/WebdriverClassicConfig.php

```diff
@@ -18,12 +17,16 @@ public static function getInstance(): self
         return new self();
     }
 
-    public function createDriver(): DriverInterface
+    public function createDriver(): WebdriverClassicDriver
     {
-        $browser = getenv('WEB_FIXTURES_BROWSER') ?: WebdriverClassicDriver::DEFAULT_BROWSER;
         $seleniumHost = $_SERVER['DRIVER_URL'];
 
-        return new WebdriverClassicDriver($browser, [], $seleniumHost);
+        return new WebdriverClassicDriver($this->getBrowserName(), [], $seleniumHost);
+    }
+
+    public function getBrowserName(): string
```

Comment:
Why this method was added?

Reply:
There's a test for the driver's `getBrowserName()` method which uses this.

It's needed in case we run the tests for other browsers (which we do already). I could replicate the function's body in the tests, but it's pointless duplication.

Reply:
The `$this->getSession()->getDriver()->getBrowserName()` code can be used instead of creating `WebdriverClassicConfig::getBrowserName` method.

Reply:
It's used for testing that method (`getDriver()->getBrowserName()`).

Using `assertSame(getDriver()->getBrowserName(), getDriver()->getBrowserName())` wouldn't make a lot of sense. 😆

That method is needed because this would not work when tests are running against e.g. firefox:
`assertSame('chrome', getDriver()->getBrowserName())`

Reply:
Ups.

I understand now what is being tested. The code is OK.